enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -110,6 +116,29 @@ Although cluster infra resources already do a best effort validation on the user
 a user could provide non-functional proxy settings/certs. This would be propagated to their Windows nodes and workloads,
 taking down existing application connectivity and preventing new Windows nodes from being bootstrapped.
 
+In case users use Windows images with PowerShell as the default shell, then there is a risk that outbound traffic 
```

Comment:
Please clarify if `Windows images` refers to a VM image, or container image

Reply:
Good catch, I replaced `images` with `nodes` to avoid confusion as the proxy configuration for Windows Workloads is out of the scope for this.